server/templates/tools/scatter.html

```diff
@@ -117,8 +130,54 @@
 
     </div>
 
-    <!-- Scatter chart -->
-    <div id="chart_div" style="width: 900px; height: 60%; margin-left: 20px; margin-top: 30px;"></div>
+    {# Scatter chart #}
+    <div id="chart-div">
+      <div id="intructions">
+        Please select two variables from the left menu, and plot locations above
+      </div>
+    </div>
+
+  </div>
+</div>
+
+{# Dialog box #}
+<div class="modal fade" id="dialog" tabindex="-1" role="dialog">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Only two variables can be selected</h5>
+        <button type="button" class="close" data-dismiss="modal"
+          aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        Check the variable you would like to replace with your new selection:
+        <div id="new-selection" class="mt-3 mb-3"></div>
+        <div class="form-check col-form-label">
+          <input class="form-check-input" type="checkbox" id="one">
+          <label class="form-check-label description" for="one" id="var-one"></label>
+        </div>
+        <div class="form-check col-form-label">
+          <input class="form-check-input" type="checkbox" id="two">
+          <label class="form-check-label description" for="two" id="var-two"></label>
+        </div>
+      </div>
+    </div>
   </div>
 </div>
-{% endblock %}
\ No newline at end of file
+
+<div id='screen'>
+  <div id="spinner"></div>
+</div>
+{% endblock %}
+
+{% block footer %}
+<script src="https://code.jquery.com/ui/1.11.1/jquery-ui.min.js"></script>
```

Comment:
do you want to remove this from the PR description?

Reply:
Removed from code - it's not required anymore, required elements are taken from bootstrap, which is included in base.html